setup.py

```diff
@@ -3,12 +3,12 @@
 from setuptools import find_packages
 
 setup(
-    name='dj-dna-streaming-python',
-    version='1.3.2',
+    name='djdp-streaming-python',
+    version='1.5.0',
```

Comment:
Probably should update major version (so 2.0.0) since we're ending python 2 support (and making breaking changes to message format in listener).